src/test/kotlin/nl/info/zac/authentication/UserPrincipalFilterTest.kt

```diff
@@ -114,7 +114,7 @@ class UserPrincipalFilterTest : BehaviorSpec({
         }
         Given(
             """A logged-in user is present in the HTTP session and a servlet request containing 
-            |a user principal with a different id as the logged-in user"""
```

Comment:
I think those `|` delimiters are useful for removing all that whitespace for new lines in test descriptions (e.g. in Codecov etc) but really not sure..